game/notes.cc

```diff
@@ -44,6 +45,17 @@ double Note::scoreMultiplier() const {
 double Note::powerFactor(double note) const {
 	if (type == FREESTYLE) return 1.0;
 	double error = std::abs(diff(note));
+	switch(config["game/difficulty"].i()){
```

Comment:
Instead of getting the integer value, might be nice to retrieve it as an enumeration for example: `Difficulty.Normal`,`Difficulty.Hard`,`Difficulty.Perfect` . This way there's no need for the `// normal mode, hard mode, perfect mode` comments and more easy to refactor in the end if needs be.